examples/response/Makefile

```diff
@@ -0,0 +1,36 @@
+# 
+# Copyright (c) 2013 No Face Press, LLC
```

Comment:
Please don't copy/paste license statements from other examples.
Use the current year and either your name or "the Civetweb developers".